### PR TITLE
Set up initial sdk-v2 and sdk-council repos

### PR DESCRIPTION
**Pull request by sentilesdal**
- update yarn.lock
- set up initial test for optimistic rewards

### CODE REVIEW

File: packages/sdk-v2/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sdk-v2",
```
Comment (DannyDelott) [inline packages/sdk-v2/package.json:2]: looks like this is a package, so `@elementfi/sdk-v2`, also we need to remove package-lock.json from this package.

File: packages/sdk-v2/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "version": "1.0.0",
+  "description": "Element SDK for V2 contracts",
+  "main": "index.js",
+  "scripts": {
+    "test": "TRANSPILE_ONLY=1 npx hardhat test"
```
Comment (DannyDelott) [inline packages/sdk-v2/package.json:7]: `TS_NODE_TRANSPILE_ONLY`

https://github.com/TypeStrong/ts-node#transpileonly

File: packages/sdk-council/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sdk-council",
```
Comment (DannyDelott) [inline packages/sdk-council/package.json:2]: @elementfi/sdk-council and remove the package-lock.json from this

File: packages/sdk-v2/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Element SDK-COUNCIL
+
+A software development kit for Element Finance council contracts
```
Comment (DannyDelott) [inline packages/sdk-v2/README.md:3]: these are switched around, this should be the core contracts, not council :)

File: packages/sdk-v2/test/proposeOptimisticRewards.ts [hunk 5]
```diff
@@ -0,0 +1,102 @@
+import {
```
Comment (DannyDelott) [inline packages/sdk-v2/test/proposeOptimisticRewards.ts:1]: This should be collocated next to its source file, not in a separate test/ directory structure.

Comment (sentilesdal) [reply packages/sdk-v2/test/proposeOptimisticRewards.ts:1]: good call

**Review by DannyDelott — APPROVED** [review]
Looks good! small request to follow the naming convention in packages, ie: `council-sdk` and `core-v2-sdk`

**Pull request merged**